src/database/queries/fixtures/canonicalFields/returnsAllCanonicalFields/seed.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO canonical_fields (
+  label,
+  short_code,
+  field_type,
+  created_at
```

Comment:
I was about to object to setting this but then again for the test purposes we need it fixed. Good call.